test/e2e_node/summary_test.go

```diff
@@ -124,7 +124,7 @@ var _ = SIGDescribe("Summary API [NodeConformance]", func() {
 				"WorkingSetBytes": bounded(10*e2evolume.Kb, memoryLimit),
 				"RSSBytes":        bounded(1*e2evolume.Kb, memoryLimit),
 				"PageFaults":      bounded(0, 1000000),
-				"MajorPageFaults": bounded(0, 10),
+				"MajorPageFaults": bounded(0, 1000),
```

Comment:
Can you remove this commit from the PR, and then we can raise this in another PR/issue? (ref https://github.com/kubernetes/kubernetes/pull/101888#discussion_r634140160)

This is changing a conformance test, and it is for making cgroup v2 tests pass, and I think that is outside the scope of this PR.

Reply:
Sure; done.